engines/scc_proxy/lib/scc_proxy/engine.rb

```diff
@@ -129,7 +140,7 @@ def announce_system_scc(auth, params)
     end
 
     def scc_activate_product(product, auth, params, mode)
-      uri = URI.parse(ACTIVATE_PRODUCT_URL)
+      uri = URI.parse(SYSTEM_PRODUCTS_URL)
```

Comment:
Shouldn't activation stay the same, i.e. the code path taken when the `registercloudguest -r $SLES_REG_CODE` is being used?

Reply:
it is the same URL, one does a post and one does a put, the URL does not mean `let's activate products` any more